controllers/binder.go

```diff
@@ -601,12 +615,11 @@ func (b *binder) remove(objs *unstructured.UnstructuredList) error {
 			return err
 		}
 
-		if b.sbr.Spec.BindAsFiles {
+		if b.sbr.Spec.BindAsFiles != nil && *b.sbr.Spec.BindAsFiles {
 			if updatedObj, err = b.removeSpecVolumes(updatedObj); err != nil {
 				return err
 			}
 		}
-
```

Comment:
are these whitespace changes needed?

Reply:
Fixed this. I do not add any extra lines in the code.. nor do I remove any line from existing code.